webapp/channels/src/packages/mattermost-redux/src/store/initial_state.ts

```diff
@@ -180,7 +180,7 @@ const state: GlobalState = {
                 bindings: [],
                 forms: {},
             },
-            pluginEnabled: true,
+            pluginEnabled: false,
```

Comment:
Also, looking at that reducer, I don't think we're going to actually set `pluginEnabled` to true because we don't update it when the config is loaded. We'd probably want to set that to true based on if the plugin is enabled by a `RECEIVED_WEBAPP_PLUGINS` action.

While this code is in `mattermost-redux` where it shouldn't be aware of plugins, I think it'd be fine to have that reducer wait for a `RECEIVED_WEBAPP_PLUGINS` and use that to set `pluginEnabled` to true

Reply:
@hmhealey Is right. If we disable the plugin via the initial state, the webapp will never know that the plugin could be enabled. Currently, we use the `/bindings/` endpoint as a "ping" to check if the plugin is enabled.